packages/devextreme/js/__internal/grids/grid_core/editing/m_editing.ts

```diff
@@ -159,16 +160,20 @@ class EditingControllerImpl extends modules.ViewController {
     this.component._optionsByReference[EDITING_CHANGES_OPTION_NAME] = true;
   }
 
-  getEditMode() {
-    const editMode: any = this.option('editing.mode');
-
+  getEditMode(): string {
```

Comment:
```suggestion
  getEditMode(): GridsEditMode {
```

https://github.com/pomahtri/devextreme/blob/82e46a958c4da11a77095399eb071461ced59267/packages/devextreme/js/common/grids.d.ts#L2313